packages/roosterjs-editor-types/lib/interface/TableFormat.ts

```diff
@@ -1,17 +1,13 @@
+import { TableBorderFormat } from '../enum/TableBorderFormat';
+
 /**
  * Table format
  */
 export default interface TableFormat {

```

Comment:
Generall, design this interface is very similar with design a database.
We need to follow database normal form such as BCNF or 3NF. We need to check each property if they are necessary, if they have conflict, if its value can be calculated from other properties. Then make sure all combinations of the values are meaningful.